src/ostree/ot-admin-builtin-unlock.c

```diff
@@ -27,7 +27,7 @@
 #include "ostree.h"
 #include "otutil.h"
 
-#include "../libostree/ostree-kernel-args.h"
+#include "ostree-kernel-args.h"
```

Comment:
We could include this through `src/libostree/ostree.h`(i.e. add  `#include <ostree-kernel-args.h>`) - that way we can omit the explicit include in this and the `src/ostree/ot-admin-instutil-builtin-set-kargs.c` files.